testsuite/features/core_ubuntu_salt.feature

```diff
@@ -1,12 +1,22 @@
-# Copyright (c) 2017-2018 SUSE LLC
+# Copyright (c) 2017-2019 SUSE LLC
 # Licensed under the terms of the MIT license.
 #
-# 1) register an Ubuntu minion via GUI
+# 0) Install salt-service
+# 1) register an Ubuntu minion via GUI (without proxy)
 # 2) run a remote command
 # 3) try an openscap scan
+# 4) Unregistre the Ubuntu minion
+# 5) Uninstall salt-service
 
 Feature: Be able to bootstrap an Ubuntu minion and do some basic operations on it
 
+#TODO: External repos are disabled, so we need to figure out how to enable the repo
+#      for salt service before uncomment it
+#@ubuntu_minion
```

Comment:
Ubuntu client tools repo will be provided for Uyuni as per https://github.com/SUSE/spacewalk/issues/6816